app/controllers/members_controller.rb

```diff
@@ -1,13 +1,17 @@
 class MembersController < ApplicationController
-  helper_method :member, :members
+  helper_method :member, :members, :talks
 
   private
 
+  def talks
```

Comment:
member_talks (more declarative, just talks - it's like all talks, might be confusing)